include/dlaf/lapack_tile.tpp

```diff
@@ -10,6 +10,18 @@
 
 #include "dlaf/common/assert.h"
 
+template <class T, Device device>
+void hegst(const int itype, const blas::Uplo uplo, const Tile<T, device>& a, const Tile<T, device>& b) {
+  DLAF_ASSERT(a.size().rows() == a.size().cols(), a.size());
+  DLAF_ASSERT(itype >= 1 && itype <= 3, itype);
+
+  // itype = 1 to solve inv(U**H)*A*inv(U) or inv(L)*A*inv(L**H))
+  // itype = 2, 3 to solve U*A*U**H or L**H*A*L
```

Comment:
IMHO this comment is superfluous, since it should be available in the documentation of the function.